Casks/pulse.rb

```diff
@@ -0,0 +1,11 @@
+cask "pulse" do
+  version "0.15.3"
+  sha256 "cfba399c46f90d11402a04137ff968ad7daf465450b5c7a3607f0f57d17904ad"
+
+  url "https://github.com/kean/Pulse/releases/download/#{version}/Pulse.app.zip"
+  name "pulse"
```

Comment:
```suggestion
  name "Pulse"
```

This is the name of the app, not a repeat of the token.

Reply:
So it is, apologies! Resolved!